faqs/how-do-participate-in-staking-on-eth-2.md

```diff
@@ -3,6 +3,7 @@ title: How do I participate in staking on ETH 2.0?
 weight: 5.0
 attribution:
   -
-    name:
-    link:
----
\ No newline at end of file
+    name: Lamboshi
+    link: https://twitter.com/L_Nakaghini
+---
+There are many ways to participate in proof of stake on Ethereum. To get started we recommend that you read the [getting started post](https://old.reddit.com/r/ethstaker/comments/jjdxvw/welcome_to_rethstaker_the_home_for_ethereum/) on [/r/ethstaker](https://old.reddit.com/r/ethstaker/). If you prefer video based content, you can check out Superphiz's [Intro to Eth2 & Staking](https://www.youtube.com/watch?v=tpkpW031RCI) ETHGlobal presentation. 
```

Comment:
- change "we recommend" to "it is recommended" - need to be careful to put words in other contributor's mouths no matter how innocuous it may be.
- hyphenate video-based